core/reflection.jvm/src/kotlin/reflect/jvm/internal/KDeclarationContainerImpl.kt

```diff
@@ -155,7 +155,7 @@ internal abstract class KDeclarationContainerImpl : ClassBasedDeclarationContain
 
         if (functions.size != 1) {
             val allMembers = members.joinToString("\n") { descriptor ->
-                DescriptorRenderer.DEBUG_TEXT.render(descriptor) + " | " + RuntimeTypeMapper.mapSignature(descriptor)
+                DescriptorRenderer.DEBUG_TEXT.render(descriptor) + " | " + RuntimeTypeMapper.mapSignature(descriptor).asString()
```

Comment:
Thanks for fixing this. Please also fix the similar code above where `RuntimeTypeMapper.mapPropertySignature` is invoked

Reply:
Thanks for pointing this out, I didn't notice the other code. Fixed.